includes/ucf-vertical-slide-assets.php

```diff
@@ -9,6 +9,8 @@ function ucf_vertical_slide_enqueue_assets() {
 
     // Enqueue scripts
     wp_enqueue_script( 'ucf-vertical-slide-script', UCF_VERTICAL_SLIDE__STATIC_URL . '/js/script.min.js', array( 'jquery' ), $version, true );
+	wp_localize_script('ucf-vertical-slide-script', 'mysiteobj', array( 'siteurl' => get_option('siteurl') ));
```

Comment:
The preferred way to get this URL is with [`get_rest_url`](https://developer.wordpress.org/reference/functions/get_rest_url/). If you want the "base" URL (so you can reuse the variable later) then just calling `get_rest_url()` will return back `{site_url}/wp-json/` which you can then add on `wp/v1/vertical_slider/` or whatever endpoint you need.

Also, we follow the fairly standard convention of capitalizing constants and since this plugin could be installed with other plugins, it's important that this constant be reliably unique. Renaming the variable to `VERTICAL_SLIDER` or something of that sort should fix this up.